template.handlebars

```diff
@@ -11,6 +11,22 @@
       {{/if}}
     </style>
 
+    {{#if @root.locals.edit}}
+      <script>
+        // preload the schemas into the cache
+        // note: this must declare all the levels, since kiln js hasn't loaded yet
+        window.kiln = {
+          services: {
+            schemaCache: {
+              {{#each @root._componentSchemas}}
+                '{{ this.name }}': {{{ stringify this.schema }}},
+              {{/each}}
+            }
+          }
+        }
```

Comment:
This works, but seems fragile. Would it be possible to use `_.set` or the slightly longer `window.kiln = window.kiln || {}; window.kiln.services = window.kiln.services || {}; ...`, or some other way to ensure nothing is overwritten?